app/views/inputBox.view.js

```diff
@@ -86,5 +67,27 @@ var MainView = Backbone.View.extend({
             ctx.fillStyle = "#FF0000";
             ctx.fillRect(rectangle3.x * size, rectangle3.y * size, rectangle3.width * size, rectangle3.height * size);
         }
+
+    },
+
+// on button click saves rectangles to database
+    saveRectangles: function () {
+        var attrsToSend = {};
+
+        for (var i = 0; i < 2; i++) {
+            for (key in this.collection.at(i).attributes) {
+                if (key !== '_id' && key !== '_createdAt' && key !== '_updatedAt') {
+                    attrsToSend[key] = this.collection.at(i).attributes[key];
+                }
+            }
+            this.collection.at(i).save(attrsToSend, {
+                    patch: true, patchAction: "update",
+                    beforeSend: function (xhr) {
+                        xhr.setRequestHeader('X-Appery-Database-Id', localStorage.getItem('X-Appery-Database-Id'),

```

Comment:
I don't think that setRequestHeader can take four parameters
https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/setRequestHeader